src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class Crypto
+    {
+        [DllImport(Interop.Libraries.CryptoNative)]
+        internal static extern IntPtr SslV2_3Method();
+
+        [DllImport(Interop.Libraries.CryptoNative)]
+        internal static extern IntPtr SslV3Method();
+
+        [DllImport(Interop.Libraries.CryptoNative)]
+        internal static extern IntPtr TlsV1Method();
+
+        [DllImport(Interop.Libraries.CryptoNative)]
+        internal static extern IntPtr TlsV1_1Method();
+
+        [DllImport(Interop.Libraries.CryptoNative)]
+        internal static extern IntPtr TlsV1_2Method();
+
+        [DllImport(Interop.Libraries.CryptoNative)]
+        internal static extern libssl.SafeSslContextHandle SslCtxCreate(IntPtr method);
```

Comment:
Yes, I wanted to leave where it is for now and move it with a separate change.
BTW - where do you think it should live?  Some SafeHandles live under Microsoft.Win32.SafeHandles, which feels weird to me for Unix code.